components/PartsList/index.js

```diff
@@ -3,31 +3,64 @@ import {
   PartsListContainer,
   StyledHeading,
   FilterButtonContainer,
+  FilterContainer,
 } from "./PartsList.styled";
 import { useState } from "react";
 
 export default function PartsList({ parts }) {
   const [categoryFilter, setCategoryFilter] = useState("alle");
+  const [statusFilter, setStatusFilter] = useState("alle");
 
   const filteredParts = parts.filter((part) => {
-    if (categoryFilter === "alle" || part.category === categoryFilter)
-      return true;
+    if (categoryFilter === "alle" || part.category === categoryFilter) {
+      if (statusFilter === "alle") {
+        return part;
+      } else if (statusFilter === "!isAssembled") {
+        return part.isAssembled === false;
+      } else if (statusFilter === "inAssembler") {
+        return part.inAssembler;
+      } else if (statusFilter === "isAssembled") {
+        return part.isAssembled;
+      } else if (statusFilter === "isSold") {
+        return part.isSold;
+      }
+    }
+    return false;
   });
 
   return (
     <>
       <StyledHeading>PARTS</StyledHeading>
-      <FilterButtonContainer>
-        <button onClick={() => setCategoryFilter("alle")}>alle</button>
-        <button onClick={() => setCategoryFilter("teller")}>teller</button>
-        <button onClick={() => setCategoryFilter("vase")}>vase</button>
-        <button onClick={() => setCategoryFilter("schüssel")}>schüssel</button>
-        <button onClick={() => setCategoryFilter("figur")}>figur</button>
-      </FilterButtonContainer>
+      <FilterContainer>
+        <FilterButtonContainer>
+          <button onClick={() => setCategoryFilter("alle")}>alle</button>
+          <button onClick={() => setCategoryFilter("teller")}>teller</button>
+          <button onClick={() => setCategoryFilter("vase")}>vase</button>
+          <button onClick={() => setCategoryFilter("schüssel")}>
+            schüssel
+          </button>
+          <button onClick={() => setCategoryFilter("figur")}>figur</button>
+        </FilterButtonContainer>
+        <FilterButtonContainer>
+          <button onClick={() => setStatusFilter("alle")}>alle</button>
+          <button onClick={() => setStatusFilter("!isAssembled")}>
+            unverbaut
+          </button>
+          <button onClick={() => setStatusFilter("inAssembler")}>
+            in Verarbeitung
+          </button>
+          <button onClick={() => setStatusFilter("isAssembled")}>
+            verbaut
+          </button>
+          <button onClick={() => setStatusFilter("isSold")}>verkauft</button>
+        </FilterButtonContainer>
```

Comment:
since there are two of the FilterButtonContainer : maybe think about making it a reusable component ;)

Reply:
@julialoeschel Do you mean like one that takes a variety of props and spits out a corresponding behaviour ?

Reply:
yes, that would be an option to avoid writing code twice. we are following DRY (dont repeat yourself). And here we see reusable code, would be a perfect component :)

Reply:
for sure i will. Later on im planning to reuse the whole filter part for an assembled-items list and maybe others. For now i will keep it

Reply:
I am a tad confused what approach you might be referring to, would it be something like:

```jsx
<FilterButtonContainer
  categoryFilters={["alle", "teller", "vase", "schüssel", "figur"]}
/>
<FilterButtonContainer
  statusFilters={["alle", "!isAssembled", "inAssembly", "isAssembled", "isSold"]}
/>
```

and have it create the buttons for each of those elements in the passed arrays ?

Reply:
added [issue//reusable FilterButtonContainer](https://github.com/matschi3/capstone/issues/11)